### ISSUE TITLE
Circle diagrams make no sense

### ISSUE THREAD

**Issue by drlemmus** [cmbi/whynot#34]
In the current implementation the circle diagram show two overlapping categories (unannotated and valid) as different sections. Just to give the hierarchy of categories:
Possible entries = missing + present
Missing = annotated + unannotated
Present = valid + obsolete

If you want to make all categories clickable, you can make two concentric circle diagrams with missing + present in the centre and with annotated + unannotated + valid + obsolete in the outer ring, aligned with the correct centre categories.

**Comment by cbaakman** [comment]
Also, databanks in why_not have parents. Like mmcif is parent of dssp.

missing: entries in parent database, but not in child database
present: entries in child database, regardless of parent database
obsolete: entries in child, but not in parent
valid: entries in child and in parent

**Comment by jonblack** [comment]
I replaced our custom pie chart code with chart.js because it's easier to maintain. The animation and interaction is just a free extra, but not so important in my opinion; certainly nice to have.

I like the idea of concentric circles, but chart.js doesn't support that. It's definitely possible with d3.js but that requires writing custom code. I've created #36 for this.

For now is it acceptable to just show `annotated`, `unannotated`, `valid`, and `obsolete` in the pie chart?
